Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -326,6 +326,11 @@ struct ContentData {
         String,
     } type { Type::Normal };
 
+    enum class HasAltText {
+        Yes,
+        No,
+    } has_alt_text { HasAltText::No };
```

Comment:
Can `alt_text` just be an `Optional<String>` instead? I don't really see the benefit of having a separate enum.